src/BuildDacpac/PackageBuilder.cs

```diff
@@ -55,6 +57,31 @@ public void AddInputFile(FileInfo inputFile)
             Model.AddObjects(File.ReadAllText(inputFile.FullName));
         }
 
+        public void AddPrePostScripts(FileInfo pre, FileInfo post, FileInfo outputFile)
```

Comment:
I think that for maintainability it might be better to have separate `AddPreDeploymentScript` and `AddPostDeploymentScript` methods here, which delegate to a private `AddDeploymentScript` method. That would lower the number of tests as well, since there are less branches to test. Then we would just call the appropriate method(s) depending on the passed arguments.